### DISCUSSION TITLE
Multiome overlays using scRNA-seq clustering

### DISCUSSION THREAD

**Discussion by millerkaplanlab** [GreenleafLab/ArchR#1808] [General]
Using Signac, I am able to select for scRNA-seq clustering and overlay with ATAC-seq data. Is there a function in ArchR that allows for this? I can't seem to find it in the manual.....

**Comment by rcorces** [comment]
I'm not sure what you mean. If by "scRNA-seq clustering" you actually mean "the embedding coordinates defined by scRNA-seq" then why not just plot that embedding and color the cells by whatever you want, for ex "PeakMatrix" or "GeneScoreMatrix"
